modules/chapter1/pages/uninstall.adoc

```diff
@@ -0,0 +1,41 @@
+= Uninstallation
+
+IMPORTANT: Before you uninstall RHDH, verify if you need to back up the data in the RHDH PostgreSQL database!. Follow the instructions in the previous section to back up the PostgreSQL database.
+
+== Uninstalling the RHDH Helm Chart
+
+You can update RHDH using the OpenShift web console.
+
+. Ensure that you are in the *Developer* perspective of the OpenShift console. Click on `Helm` to view the installed releases in the `Helm Releases` tab.
+
+. Expand the hamburger menu on the right of the `rhdh` release and click `Delete Helm Release`.
++
+image::helm-release-delete.png[title=Delete Helm Release]
+
+. Type the release name `rhdh` into the dialog that pops up and click `Delete` to delete the resources created by the helm chart.
+
+. The secret containing the encrypted Quay.io access token is not deleted automatically by the helm chart deletion process. If you do not need this secret anymore, delete the `rhdh-pull-secret` from the `Secrets` menu in the `Developer` perspective.
+
+== Uninstalling the RHDH Helm Chart using the CLI
+
+You can uninstall RHDH using the *helm* CLI.
+
+. Log in to the OpenShift cluster as the cluster administrator. Ensure that you switch to the *devhub* project.
++
+```bash
+$ oc login -u admin https://api.cluster-<your_guid>.dynamic.opentlc.com:6443
+$ oc project devhub
+```
+
+. Run the following command to uninstall RHDH
++
+```bash
+$ helm uninstall 
```

Comment:
I got an error here saying it requires at least 1 argument.
Had to pass it `rhdh` like this:

`helm uninstall rhdh`

Reply:
fixed